libeos-updater-util/flatpak.c

```diff
@@ -121,6 +121,18 @@ euu_flatpak_remote_ref_action_unref (EuuFlatpakRemoteRefAction *action)
   g_slice_free (EuuFlatpakRemoteRefAction, action);
 }
 
+char *
```

Comment:
Nitpick: The eos-updater coding style uses `gchar` rather than `char`.

Reply:
Please add gtk-doc comments for these new functions, too, since this is now essentially a public library, so we should be working towards full documentation coverage.

Reply:
Also, the indentation is wrong inside these functions.